src/vs/workbench/api/common/extHostSearch.ts

```diff
@@ -88,6 +89,12 @@ export class ExtHostSearch implements ExtHostSearchShape {
 		}
 	}
 
+	$doInternalFileSearchWithCustomCallback(query: IFileQuery, token: CancellationToken, handleFileMatch: (data: UriComponents[]) => void): Promise<ISearchCompleteStats> {
+		return Promise.resolve({
```

Comment:
There's rarely a reason to use promise.resolve in 2023, much simpler to use async/await